easybuild/easyblocks/s/scipy.py

```diff
@@ -211,4 +223,12 @@ def sanity_check_step(self, *args, **kwargs):
             'dirs': [det_pylibdir()],
         }
 
-        return PythonPackage.sanity_check_step(self, custom_paths=custom_paths)
+        # make sure that scipy is included in output of 'pip list',
+        # so that 'pip check' passes if scipy is a required dependency for another Python package;
+        # use case-insensitive match, since name is sometimes reported as 'SciPy'
+        custom_commands = [
+            "pip list",
+            "pip list | grep -i '^scipy\s*%s\s*$'" % self.version,
```

Comment:
```suggestion
            "pip list | grep -i r'^scipy\s*%s\s*$'" % self.version,
```

Reply:
fixed in [de55fd4](https://github.com/easybuilders/easybuild-easyblocks/pull/2903/commits/de55fd4f28aaf79b3eec21c6c433e592c79c8219) (`\s` was also wrong, must be `[ \t]` since it's `grep`, not Python regex...)